files/jira.rb

```diff
@@ -32,9 +49,18 @@ def create_issue(summary, full_description, project)
             ]
           }
         }
-        issue.save(issue_json)
-        url = get_options[:site] + '/browse/' + issue.key
-        puts "Created issue #{issue.key} at #{url}"
+        # FIXME - would this work
+        components = []
+        if components.size?
+          issue_json['fields']['components'] = components
```

Comment:
We added a param to our moncheck module,  "tags" and we use that for every handler really. It's just an array you can pass that is used in the JIRA handler for labels and any other handler.

I guess the components here is for different JIRA projects using different fields etc, so it might be nice in this case.